### ISSUE TITLE
Feature Request: Option to ignore merge commits for GPG check

### ISSUE THREAD

**Issue by rallytime** [jarrodldavis/probot-gpg#41]
First of all, thanks for building this GPG check for verified commits. This is going to be helpful for our team! ❤️ 

We've enabled the GPG Probot on the [salt repo](https://github.com/saltstack/salt) and have realized that the GPG check errors when merge commits are created from the GitHub web interface. 

As an example, we have `Require branches to be up to date before merging` option enabled on our branches, which allows us to use the `Update Branch` option. The problem is that when you use this button to keep the PR up to date with the HEAD of the base branch, the merge commit can't be verified. 

Here's an example PR: https://github.com/saltstack/salt/pull/43707 

We can't simultaneously keep the PR's branch up to date AND have all of the commits be in compliance with the GPG check. 

Would it be possible to add some kind of flag to ignore merge commits from the web UI for the GPG check?

**Comment by jarrodldavis** [comment]
I can certainly look into providing options like [probot/stale](https://github.com/probot/stale#usage) and [behaviorbot/request-info](https://github.com/behaviorbot/request-info#usage) do using a yaml file in the repository under the `.github` repository. Would that work for you?

**Comment by rallytime** [comment]
@jarrodldavis Yes, that would be wonderful!

**Comment by rallytime** [comment]
Hi @jarrodldavis - I noticed today that GitHub has started marking merge commits from the web UI as `Verified`. I wanted to mention it here in case that is helpful for you. I am not sure if a config with this flag is warranted still or not on your end, or if it is just a matter detecting a different kind of verification? (I have not looked at the code here to see how this works, or if the verification designation from GitHub is different somehow.)

Here is an example: https://github.com/saltstack/salt/pull/44311

The last commit there is marked as `Verified`, even though I all I did was click the `Update Branch` button from the Web UI, just as before.

**Comment by jarrodldavis** [comment]
Whoa neat it looks like @web-flow has its own GPG key. Thanks for the heads up!

It looks like the other bug (#59) is still occurring, or maybe there's some different information in the API for verified web-flow commits that the app isn't expecting. I'll keep investigating, especially once #45 is completed.

**Comment by rallytime** [comment]
Yeah! It looks like it is working now. I think perhaps when it was failing before it is hitting the other bug, as you suggested. Here's a merge commit with the green status for the GPG check: https://github.com/saltstack/salt/pull/44265.
